app/models/Sprint.php

```diff
@@ -92,4 +98,36 @@ private function createPhabricatorProject()
 
 		return true;
 	}
+
+	/**
+	 * @return bool Returns whether the snapshot was successfully created
+	 */
```

Comment:
Do you need to do this return for some reason? It's a classical anti-pattern violating the [CQS principle](http://en.wikipedia.org/wiki/Command%E2%80%93query_separation)